scripts/build.sh

```diff
@@ -36,16 +32,9 @@ if [ -z "${SKIP_CARGO_BUILD:-}" ]; then
 fi
 export PATH="$PWD/target/${CARGO_PROFILE:-debug}:$PATH"
 
-
-# Function for killing processes stored in FM_PID_FILE in reverse-order they were created in
-function kill_fedimint_processes {
-  echo "Killing fedimint processes"
-  PIDS=$(cat $FM_PID_FILE | sed '1!G;h;$!d') # sed reverses order
-  if [ -n "$PIDS" ]
-  then
-    kill $PIDS 2>/dev/null
-  fi
-  rm -f $FM_PID_FILE
+function kill_devimint {
+  echo "Killing devimint and child processes"
+  pkill devimint
```

Comment:
What if I'm running multiple devimints at the same time? :P

Reply:
Maybe it should just be a pid file for devimint?

Reply:
I don't think devimint supports multiple concurrent sessions. 

If we're going to have a PID file, I suggest devimint manages it and not shell scripts. Currently you need to copy a bunch of shell scripts in order to use devimint in external repository which sucks.

Reply:
Fedimint should manage it, yes.